--- ./node_modules/@mapbox/react-native-mapbox-gl/android/rctmgl/build.gradle	2019-03-15 22:08:11.000000000 -0500
+++ ./node_modules/@mapbox/react-native-mapbox-gl/android/rctmgl/build.gradle	2019-03-15 22:05:54.000000000 -0500
@@ -28,9 +28,12 @@
     // Mapbox SDK
     compile 'com.mapbox.mapboxsdk:mapbox-android-services:2.2.9'

-    compile('com.mapbox.mapboxsdk:mapbox-android-sdk:5.4.1@aar') {
-        transitive=true
-    }
+    implementation 'com.android.support:support-vector-drawable:28.0.0'
+    implementation 'com.android.support:support-annotations:28.0.0'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.12.1'
+
+    implementation 'com.mapbox.mapboxsdk:mapbox-android-sdk:5.5.1@aar'

     // Mapbox plugins
     compile 'com.mapbox.mapboxsdk:mapbox-android-plugin-localization:0.1.0'
