diff --git node_modules/@callstack/react-theme-provider/typings/index.d.ts node_modules/@callstack/react-theme-provider/typings/index.d.ts
index c209bec..228fb92 100644
--- node_modules/@callstack/react-theme-provider/typings/index.d.ts
+++ node_modules/@callstack/react-theme-provider/typings/index.d.ts
@@ -8,7 +8,7 @@ type $Without<T, K extends keyof any> = T extends any ? Pick<T, Exclude<keyof T,
 type $DeepPartial<T> = { [P in keyof T]?: $DeepPartial<T[P]> };

 export type ThemingType<Theme> = {
-  ThemeProvider: React.ComponentType<{ theme?: Theme }>;
+  ThemeProvider: React.ComponentType<React.PropsWithChildren<{ theme?: Theme }>>;
   withTheme: <Props extends { theme: Theme }, C>(
     WrappedComponent: React.ComponentType<Props> & C
   ) => React.ComponentType<
