--- ./node_modules/@callstack/react-theme-provider/lib/createTheming.js.flow	2019-03-16 15:00:11.000000000 -0500
+++ ./node_modules/@callstack/react-theme-provider/lib/createTheming.js.flow	2019-03-16 15:00:54.000000000 -0500
@@ -11,6 +11,7 @@ import type { $DeepShape } from './types';
 export type ThemingType<T> = {
   ThemeProvider: ThemeProviderType<T>,
   withTheme: WithThemeType<T>,
+  useTheme(overrides?: $DeepShape<T>): T,
 };

 export default function createTheming<T: Object>(
