diff --git node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
index 4ff336234..c139440cc 100644
--- node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
+++ node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
@@ -41,7 +41,7 @@
   YGNodeMarkDirty(self.yogaNode);
 }
 
-static YGSize RNDateTimePickerShadowViewMeasure(YGNodeConstRef node, float width, YGMeasureMode widthMode, float height, YGMeasureMode heightMode)
+static YGSize RNDateTimePickerShadowViewMeasure(YGNodeRef node, float width, YGMeasureMode widthMode, float height, YGMeasureMode heightMode)
 {
   RNDateTimePickerShadowView *shadowPickerView = (__bridge RNDateTimePickerShadowView *)YGNodeGetContext(node);
