diff --git ios/Pods/Sentry/Sources/Sentry/include/SentryThreadMetadataCache.hpp ios/Pods/Sentry/Sources/Sentry/include/SentryThreadMetadataCache.hpp
index 3d7b2fe..b198343 100644
--- ios/Pods/Sentry/Sources/Sentry/include/SentryThreadMetadataCache.hpp
+++ ios/Pods/Sentry/Sources/Sentry/include/SentryThreadMetadataCache.hpp
@@ -66,7 +66,7 @@ class ThreadMetadataCache {
         struct ThreadHandleMetadataPair {
             ThreadHandle::NativeHandle handle;
             ThreadMetadata metadata;
         };
-        std::vector<const ThreadHandleMetadataPair> threadMetadataCache_;
-        std::vector<const QueueMetadata> queueMetadataCache_;
+        std::vector<ThreadHandleMetadataPair> threadMetadataCache_;
+        std::vector<QueueMetadata> queueMetadataCache_;
     };
