--- ./node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+++ ./node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
@@ -767,7 +767,7 @@
 #endif
 }
 
-- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<id<RCTBridgeModule>> *)modules
+- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<Class> *)modules
                                withDispatchGroup:(dispatch_group_t)dispatchGroup
                                 lazilyDiscovered:(BOOL)lazilyDiscovered
 {

--- ./node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm
+++ ./node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm
@@ -297,7 +297,7 @@
           @"%@ has no setter or ivar for its bridge, which is not "
            "permitted. You must either @synthesize the bridge property, "
            "or provide your own setter method.",
-          RCTBridgeModuleNameForClass(module));
+          RCTBridgeModuleNameForClass(Class(module)));
     }
   }
 

--- ./node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
+++ ./node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
@@ -270,6 +270,9 @@
     layer.contentsScale = self.animatedImageScale;
     layer.contents = (__bridge id)_currentFrame.CGImage;
   }
+  else {
+    [super displayLayer:layer];
+  }
 }
 
 #pragma mark - Util
